source/direct_debit/index.html.md.erb

```diff
@@ -241,6 +241,13 @@ You could then get a single page of these mandates by running:
 
 This would return payments 501 to 1000.
 
+## Test a mock bank account
+
+To test confirming bank account details as a user, use the following with your test account:
+
+- `200000` as the sort code
```

Comment:
Maybe write the sort code as “20 00 00”. The style guide at https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#bank-details says to put spaces in sort codes and we accept that format on the payment pages.

Reply:
Ah good point! I'll fix that, thanks Alex.